index.html

```diff
@@ -26,17 +26,17 @@
         </ul>
         <ul class="navigation-list">
           <li class="navigation-items">
-            <a class="navigation-link" href="#">Каталог товаров</a>
+            <a class="navigation-link-menu" href="#">Каталог товаров</a>
             <button class="button" type="submit">Подписаться</button>
```

Comment:
```suggestion
            <button class="button" type="button"></button>
```
А также в данной кнопке текста нет. Там только изображение. Следовательно нужно добавить в кнопку поясняющий текст, который будет скрытым.